src/js/components/Diagram/StyledDiagram.js

```diff
@@ -38,14 +33,16 @@ const animationStyle = props => {
   `;
 };
 
+const availableAnimations = [true, 'draw', 'pulse'];
+
 const connectionStyle = (connections, theme) =>
   connections.map((connection, index) => {
     if (connection !== undefined && connection.props.animation !== undefined) {
       const { type } = connection.props.animation;
       const animationType = type || connection.props.animation;
 
       if (!availableAnimations.includes(animationType)) {
```

Comment:
Similar to the above comment -- should we only call connectionStyle if the animationType is included in the available animations array?

Reply:
Gonna take a minute, but I'll see what I can come up with!